src/dnddiceroller/Ordo_DiceTower.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package ordo_dicetower;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author zacharyzordo
+ */
+public class Ordo_DiceTower {
+
+    /**
+     * @param args the command line arguments
+     */
+    
+    public static void main(String[] args) {
+        boolean d6Pass = true;
+        boolean towerPass = true;
+	for(int i = 0; i< 1000; i++){
+            int d6 = test_oneD6();
+            if(d6 != -1){
+		System.out.println("Die Test Failed with Value: " + d6);
+                d6Pass = false;
+            }
+            int tower = test_diceTowerWithTwoD6();
+            if(tower != -1){
+                System.out.println("Tower Test Failed with Value: " + tower);
+                towerPass = false;
+            }   
+        }
+        if(d6Pass){
+            System.out.println("Die Test Passed");
+        }
+        if(towerPass){
+            System.out.println("Tower Test Passed");
+        }
+    }
+  
+    public static int test_oneD6(){
+	Die die = new Die(6);
+	die.roll();
+	int dieValue = die.getDieValue();
+	if(dieValue >= 1 && dieValue <= 6){
+            return -1; // Means the die value is outside its bounds for a d6
+	}else{
+            return dieValue;
+	}
+    }
+  
+    public static int test_diceTowerWithTwoD6(){
+	List<Die> dice = new ArrayList();
+	dice.add(new Die(6));
+	dice.add(new Die(6));
+	DiceTower tower = new DiceTower(dice);
+	tower.whatAreTheDiceValues();
+	int trayValue = tower.trayValue();
+	if(trayValue >= 2 && trayValue <= 12){
+            return -1; // means the die value is outside the bounds of 2 d6
+	}else{
+            return trayValue;
+	}
+    }
+}
+
```

Comment:
I agree with frosch. Adding the output to let you know it passed the test is a nice touch I should have added this to mine. Other than that I'd removing this commented out code just to make it look cleaner. Unless you had a good reason for saving it there!